src/erllambda.erl

```diff
@@ -325,11 +326,11 @@ invoke_exec( Handler, Event, Context ) ->
             {unhandled, Response}
     end.
 
-invoke_credentials( #{<<"AWS_SESSION_TOKEN">> := Token} = _Context ,
-                    {ok, #aws_config{security_token = Token}}) ->
+invoke_credentials( #{<<"AWS_SESSION_TOKEN">> := Token1} = _Context ,
```

Comment:
what is the purpose of this change?

Reply:
In the code
```
Context = maps:merge(os_env2map(), hdr2map(EventHdrs)),
os_env2map() ->
    maps:from_list(lists:map(
        fun(S) ->
            [K, V] = string:split(S, "="),
            {list_to_binary(K), list_to_binary(V)}
        end,
        os:getenv()
    )).
hdr2map(Hdrs) ->
    maps:from_list([{list_to_binary(K), list_to_binary(V)} || {K, V} <- Hdrs]).
```
The `Context` variable is a map, which key type is binary, and its value is also binary.
in the `erlcloud/include/erlcloud_aws.hrl` head file:
```
security_token=undefined::string()|undefined,
```
So, `Token1` is binary, and `Token2` is `string()` or `undefined`, it never euqals.
Just changing the type of writing to ignore the dialyzer error.

Reply:
this looks like a bug, and should be fixed, rather than hidden.